wagtail/wagtailimages/tests/test_image_operations.py

```diff
@@ -1,4 +1,4 @@
-import unittest
+from django.test import TestCase
```

Comment:
Out of interest, what's the reason for this switch? I chose ``unittest`` as these tests don't need database rollback